### PR TITLE
SKU of seats added in sailthru content

### PR DESCRIPTION
**Pull request by tasawernawaz**
@ahsan-ul-haq @schenedx Please review this.

### CODE REVIEW

File: sailthru_content/services/sailthru_translation_service.py [hunk 1]
```diff
@@ -97,6 +97,16 @@ def _get_tags_from_property(self, source_array, key, tag_prefix):
 
         return tags
 
+    def _return_sku(self, seats):
+        """ Return dict of sku for a course run. """
+        sku = {}
```
Comment (clintonb) [inline sailthru_content/services/sailthru_translation_service.py]: Every seat type has a SKU. Why the limitations?

`return {seat['type']: seat['sku'] for seat in seats}`

Comment (tasawernawaz) [inline sailthru_content/services/sailthru_translation_service.py]: It was a requirement that we only need `sku` of `verified` or `professional` seats.

File: sailthru_content/services/sailthru_translation_service.py [hunk 2]
```diff
@@ -97,6 +97,16 @@ def _get_tags_from_property(self, source_array, key, tag_prefix):
...
+        for seat in seats:
+            seat_type = seat.get('type')
+            if seat_type in ['verified', 'professional']:
```
Comment (schenedx) [inline sailthru_content/services/sailthru_translation_service.py]: Can you set the array `['verified', 'professional']` as a constant somewhere in this file?

File: sailthru_content/services/sailthru_translation_service.py (hunk 1 above)
Comment (schenedx) [inline sailthru_content/services/sailthru_translation_service.py]: You can actually use the one liner suggested by @clintonb with the if statement.

File: sailthru_content/services/sailthru_translation_service.py [hunk 3]
```diff
@@ -133,6 +143,10 @@ def translate_course_run(self, course_run, course, program_dictionary=None):
         if len(tags) > 0:
             sailthru_content['tags'] = ", ".join(tags)
 
+        seats = course_run.get('seats')
+        if seats:
```
Comment (schenedx) [inline sailthru_content/services/sailthru_translation_service.py]: I thought dictionary `{}` would still evaluate to `True`?

Comment (tasawernawaz) [inline sailthru_content/services/sailthru_translation_service.py]: `seats` is a empty list `[ ]` and will evaluates `True` only when there is at least one `seat` type exists.

File: sailthru_content/services/sailthru_translation_service.py (hunk 2 above)
Comment (tasawernawaz) [inline sailthru_content/services/sailthru_translation_service.py]: added.

File: sailthru_content/services/tests/test_sailthru_translation_service.py [hunk 4]
```diff
@@ -103,8 +104,13 @@ def _get_expected_sailthru_item(self, seat_type=None):
             'date': '2016-09-23'
         }
 
+        if seat_type in ['professional', 'verified']:
```
Comment (schenedx) [inline sailthru_content/services/tests/test_sailthru_translation_service.py]: You can just replace this list with the `sailthru_translation_service.REQUIRED_SEATS_TYPES`

File: sailthru_content/services/tests/test_sailthru_translation_service.py [hunk 5]
```diff
@@ -70,7 +71,7 @@ def test_translate_programs(self):
         self.assertEqual(len(translated_programs[0]['vars']['course_runs']), 4)
 
     def _get_expected_sailthru_item(self, seat_type=None):
```
Comment (clintonb) [inline sailthru_content/services/tests/test_sailthru_translation_service.py]: This method doesn't return anything.

Comment (clintonb) [inline sailthru_content/services/tests/test_sailthru_translation_service.py]: It seems we have tests; but, they aren't actually being run by Travis. That should be fixed.

Comment (clintonb) [inline sailthru_content/services/tests/test_sailthru_translation_service.py]: Got it. The test is still broken.

**Pull request merged**